docs/community/zkillboard/index.md

```diff
@@ -0,0 +1,21 @@
+---
+search:
+  exclude: true
+
+title: zkillboard.com
+type: service
+description: zkillboard.com is a killboard for the Massively Multiplayer Online Role Playing Game (MMORPG) EVE-Online.
+maintainer:
+  name: Squizz Caphiantor
```

Comment:
You need to add a GitHub username, otherwise this won't work. The `name` become the link text and `github` becomes a GitHub profile link. I also think you made a typo in your own name, but you are the authority on that. :smile: 

```suggestion
maintainer:
  name: Squizz Caphiantor
  github: cvweiss
```